packages/strategy-tests/src/lib.rs

```diff
@@ -323,6 +323,71 @@ impl PlatformDeserializableWithPotentialValidationFromVersionedStructure for Str
     }
 }
 
+pub type MempoolDocumentCounter<'c> = &'c BTreeMap<(Identifier, Identifier), u64>;
```

Comment:
It's more convenient to define type without reference so you can use it with and without reference: `MempoolDocumentCounter ` and `&MempoolDocumentCounter`.

Reply:
Ok